src/app/modules/authController.js

```diff
@@ -14,16 +30,42 @@ class UserController {
       password: hashPassword,
     };
     const result = await UserServices.create(authData);
-    const { userId } = result.dataValues;
-    const token = jwt.createToken(userId);
 
+    const uID = req.body.username;
+    const getId = await UserServices.findUserid(uID);
+
+    const { id } = getId;
+    const token = jwt.createToken(id);
     if (!result) {
       return GenericHandler.error(res, 500, 'incorrect method');
     }
     return GenericHandler.success(res, {
-      userId, email, username, token,
+      id, email, username, token,
     }, 201, 'user succesfuly registed');
   }
+
+  static async signin(req, res) {
+    const { username, password } = req.body;
+    const checkUser = req.body.username;
+    const getUser = await UserServices.signinUser(checkUser);
+
+    if (!getUser) {
+      const resp = GenericHandler.error(res, 404, 'user does not exist');
+      console.log(resp);
+      return resp;
+    }
+
+    const realPass = Password.passCompare(
+      password,
+      getUser.password,
+    );
+
+    if (!getUser || !realPass) {
+      GenericHandler.error(res, 404, 'incorrect credentials');
```

Comment:
This looks good to me. Just one thing to improve.
- Create a file that holds all the response statuses and response messages.

Reply:
okay thanks 
but can i do after or before merging ?

Reply:
First, make the changes then merge.

Reply:
okay got it